app/models/event.rb

```diff
@@ -64,12 +66,11 @@ def respondents
   end
 
   def eligible_people
-    Person.active.all # In the future, this will need to honor department
-    # def eligible_people
-    #   self.departments.each do |department|
-    #
-    #   end
-    # end
+    eligible_people = []
```

Comment:
I'm ok with that edge case, which I think would also be present in my implementation. 
I played around a whlie trying to get this into a one liner, but kept coming at it from the event, not Person. Thanks ! I'm going to go try it.